packages/widget-chat/src/components/activity-item-share-list/index.js

```diff
@@ -0,0 +1,49 @@
+import React, {PropTypes} from 'react';
+import classNames from 'classnames';
+
+import injectFileDownloader from '../../containers/file-downloader';
+import ActivityItemBase from '../activity-item-base';
+import ShareFile from '../activity-item-share-file';
+import ShareThumbnail from '../activity-item-share-thumbnail';
+import styles from './styles.css';
+
+function ActivityItemShareList(props) {
+  const {
+    files,
+    onDownloadClick,
+    share
+  } = props;
+
+  const items = files.map((file) => {
+    if (file.image) {
+      const thumbnail = file.mimeType === `image/gif` ? share.files[file.url] : share.files[file.image.url];
```

Comment:
In that case, well done!
![img](http://media4.giphy.com/media/pA7VTH97OH7Ko/giphy.gif)